.travis.yml

```diff
@@ -106,7 +96,7 @@ matrix:
        TARGET_64=x86_64-pc-windows-msvc
        TARGET_32=i686-pc-windows-msvc
      before_install:
-      - choco install golang nasm
```

Comment:
can drop nasm too? I think its a BSSL dep

Reply:
No, we still need this to actually build the ASM files.